modules/plugin/imagepyramid/src/main/java/org/geotools/gce/imagepyramid/ImagePyramidFormat.java

```diff
@@ -206,20 +206,15 @@ public boolean accepts(Object source, Hints hints) {
             }
```

Comment:
You are right, that piece of code doing custom parsing trick me. The params are directly passed down as-is. I have updated the PR.